polkadot/runtime/common/src/paras_registrar/migration.rs

```diff
@@ -56,14 +58,24 @@ impl<T: Config, UnlockParaIds: Contains<ParaId>> OnRuntimeUpgrade
 		let new_count = Paras::<T>::iter_values().count() as u32;
 
 		ensure!(old_count == new_count, "Paras count should not change");
-		Ok(())
+
+		Paras::<T>::iter_keys().try_for_each(|para_id| -> Result<(), _> {
+			let info = Paras::<T>::get(para_id).unwrap();
+			ensure!(
+				info.billing_account.is_none(),
+				"The billing account must be set to the para manager"
```

Comment:
It must be `None`, but it also must be set to the para manager?

Reply:
Yeah, this must be `None`, my bad.
```suggestion
				"The billing account must be set to the None"
```